errutil/error_test.go

```diff
@@ -0,0 +1,42 @@
+package errutil_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/madlambda/spells/errutil"
+)
+
+func TestErrorSentinelWrapping(t *testing.T) {
+	const (
+		someError        errutil.Error = "someError"
+		someAnotherError errutil.Error = "someAnotherError"
+	)
+
+	wrappedErr := fmt.Errorf("wrapping up: %w", someError)
+	assertErrorIsWrapped(t, wrappedErr, someError)
+
+	wrappedErr2 := fmt.Errorf("wrapping up: %w", someAnotherError)
+	assertErrorIsWrapped(t, wrappedErr2, someAnotherError)
+}
+
+func TestErrorRepresentation(t *testing.T) {
+	const (
+		someError errutil.Error = "someError"
+	)
+
+	got := someError.Error()
+	want := string(someError)
+
+	if got != want {
```

Comment:
One of the exceptions where the string representation of the error is actually part of the type interface, I want to guarantee that whatever string you used to initialized the type will be verbating on the error Error() return, so you can count on that when building you errors (wrapping/chaining/etc).